pages/services.js

```diff
@@ -93,6 +97,95 @@ const Services = () => (
           },
         ]}
       />
+      <Layout nested={true} type="2-1" border={false}>
+        <div>
```

Comment:
Diego, nice work! But since this div doesn't have a class or an attribute, wouldn't be better to replace this `<div>` with a React fragment `<>`?

Reply:
Yeah, you are right. I will make this change.